### ISSUE TITLE
typescript definitions

### ISSUE THREAD

**Issue by xenoterracide** [decompil3d/servicemeow#27]
was going to give this a try for querybuilder, but seems there are no type definitions

**Comment by decompil3d** [comment]
@xenoterracide The typings are defined in TS-compatible JSDoc, so Typescript should be able to pick them up. Let me know if you find cases where that doesn't hold up.

**Comment by xenoterracide** [comment]
lovely but, no

```
TSError: ⨯ Unable to compile TypeScript:
src/specifications/service-now/service-now.spec.ts(5,30): error TS7016: Could not find a declaration file for module 'servicemeow'. '/Users/nqy642/IdeaProjects/exceptionsone/node_modules/servicemeow/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/servicemeow` if it exists or add a new declaration (.d.ts) file containing `declare module 'servicemeow';`
 ```
 
 ```
 import { QueryBuilder } from 'servicemeow'

import { ServiceNowQueryBuilder } from './service-now'

type TestType = {
  a: string
}

describe('ServiceNowSpecification', () => {
  new QueryBuilder()
```

```
npm explain servicemeow
servicemeow@3.0.0
node_modules/servicemeow
  servicemeow@"^3.0.0" from back-end@0.0.0
  back-end
    back-end@0.0.0
    node_modules/back-end
      workspace back-end from the root project
```

you actually have to provide the `.d.ts`

**Comment by decompil3d** [comment]
Fixed in #28 -- update to version 3.1 😄 

It auto-generates typings from the JSDoc at publish time, so `*.d.ts` files are in the npm package, but not git.
